java/src/IceGridGUI/src/main/java/com/zeroc/IceGridGUI/SessionKeeper.java

```diff
@@ -371,6 +376,8 @@ public void registerObservers() throws Throwable {
         private final AdminSessionPrx _session;
         private final boolean _routed;
 
+        private java.util.concurrent.Future<?> _keepAliveFuture;
```

Comment:
Something is still missing after this revert - keepAliveFuture isn't assigned anywhere.

Reply:
I think this was leftover dead code. I will remove it in a follow-up PR.

Reply:
In the PR you reverted, there is a comment that this was a leftover from a previous PR. It is never set in the 3.7 branch.